easybuild/easyconfigs/c/cget/cget-0.1.6-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'Bundle'
+
+name = 'cget'
+version = '0.1.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+exts_defaultclass = 'PythonPackage'
```

Comment:
@jdemeul Please move this down right above the `exts_list = ` line, and also include this (we want to start migrating to this by default for Python packages):

```python
exts_default_options = {
    'download_dep_fail': True,
    'use_pip': True,
}
```

This i) checks whether additional Python packages are downloaded & installed automatically (which we don't like, since then we don't control the versions), ii) uses `pip` rather than the default `setup.py install` procedure to install the Python packages (which is the recommended way to install Python packages).

So, you'll need to re-test this easyconfig and potentially add other Python packages (in he right order).